examples/js/shaders/VolumeShader.js

```diff
@@ -0,0 +1,320 @@
+// Author: Almar Klein
+// Description: Shaders to render 3D volumes using raycasting.
+// The applied techniques are based on similar implementations in the Visvis and Vispy projects.
+// This is not the only approach, therefore it's marked 1.
+
+
+THREE.ShaderLib[ 'volumerender1' ] = {
```

Comment:
We don't add shader code into `THREE.ShaderLib` from `examples/js/shaders`. Refer to the existing shader files in `examples/js/shaders/`. Perhaps `THREE.VolumeRender(ing)` would looks better?

Reply:
I just realized Ocean example adds into `THREE.ShaderLib` now. But I think we generally don't from `examples`. Perhaps we need to update Ocean example...

Reply:
done

Reply:
Ha, then I may just have been unlucky, because that was indeed one of the examples that I looked at :)